### PR TITLE
Add support for RHEL 8 based distros

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: manifests/repo/nodesource.pp [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
   case $facts['os']['family'] {
     'RedHat': {
-      if $facts['os']['release']['major'] =~ /^[67]$/ {
+      if $facts['os']['release']['major'] =~ /^[679]$/ {
```
Comment (ghoneycutt) [inline manifests/repo/nodesource.pp]: I believe you meant to use `678` and not `9`

**Comment by ghoneycutt** [comment]
Thanks, great looking PR! Could you also add support to the README under https://github.com/voxpupuli/puppet-nodejs#limitations

**Comment by mmoll** [comment]
There are still some problems, at least the one with the `nodejs` package from Nodesource can also be seen on real systems (problem is IMHO in the NodeJS RPM itself), for EPEL, I'll need to have a look...

**Comment by ghoneycutt** [comment]
Appears to trying to load a template for EPEL8 which does not exist.

**Comment by mmoll** [comment]
https://github.com/stahnma/puppet-module-epel/pull/92 is waiting for a new module release.

**Comment by mmoll** [comment]
Fot the nodesource repo to work, `yum module disable -y nodejs` would be needed (see https://github.com/nodesource/distributions/pull/997). Is there a puppet way to do this?

**Comment by dhoppe** [comment]
@mmoll As far as I know ```puppet/yum``` or ```puppetlabs/yumrepo_core``` do not support the handling of modules. 

You could use the template resource to create the following file:
```
[root@1ca0b18b505b ~]# cat /etc/dnf/modules.d/nodejs.module
[nodejs]
name=nodejs
stream=
profiles=
state=disabled
```

**Comment by mmoll** [comment]
@dhoppe Thanks for that hint, seems to work well. :bowing_man: 

This should be ready for review/merge then.

**Pull request merged**